go.mod

```diff
@@ -1,6 +1,6 @@
 module k8s.io/publishing-bot
 
-go 1.20
+go 1.22.0
```

Comment:
why not 1.23?

Reply:
Publishing bot repo is now used in https://github.com/kubernetes/kubernetes/blob/master/hack/tools/go.mod, as part of https://github.com/kubernetes/kubernetes/pull/125801. So when we bumb to 1.23 here the hack/tools also will have to be bumped to 1.23